content/general/functions.textile

```diff
@@ -28,6 +187,13 @@ We can support the following server-less platforms:
 * Google Cloud Function
 * Azure Functions
 
-h2. Getting started
+We also offer Custom Endpoints to our "Enterprise":https://www.ably.io/pricing/enterprise customers.
+
+h2(#troubleshooting). Troubleshooting Functions
+
+If you are finding it hard to debug a problem with Reactor Functions, we recommend you review "our recommendations for debugging Functions":https://support.ably.io/solution/articles/3000074193-how-can-i-debug-reactor-function. If this does not help, please "get in touch with us":https://www.ably.io/contact so that we can help you with your problem.
```

Comment:
We need to add a screenshot (once you have completed your work wtih Diogo) of a failed reactor rule and be explicit about where it appears in your console (i.e. not in the channel info)